src/index.js

```diff
@@ -17,15 +17,15 @@ import "./css/app.scss"
 // Components
 import { App, Home } from "./components"
 
-import {
-  Start as StartDeposit,
-  Invoice,
-  GetAddress,
-  Pay,
-  Confirming as ConfirmingDeposit,
-  Prove as ProveDeposit,
-  Congratulations as CongratulationsDeposit,
-} from "./components/deposit"
+// import {
```

Comment:
Why are we commenting out the code but not removing it? If we want to restore the features we can always refer to this PR.

Reply:
c3a3325 and 067297b